src/components/SiteHeader/index.js

```diff
@@ -90,6 +90,16 @@ const HeaderWrapper = styled.div`
       #header {
         position: static;
 
+        header {
+          height: 160px;
+        }
+
+        @media screen and (max-width: 539px) {
```

Comment:
Do we have breakpoint variables for this?

Reply:
No, there aren't any defined for the whole application. Most breakpoints are defined by `@datapunt/asc-ui`'s theme configuration. Some others, where applicable, are defined on component level. Most of the time, that makes most sense.